Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -1,30 +1,23 @@
-
 function HandleCodeAnalysisReporting
 {	
     Write-Verbose "HandleCodeAnalysisReporting"
     if (IsPrBuild)
     {   
-        Write-Host "Fetching code analysis issues and posting them to the PR..."  
-         
+        Write-Host "Fetching code analysis issues and posting them to the PR..."
+           
         Import-Module -Name "$PSScriptRoot/ReportProcessor-Module.psm1"
         Import-Module -Name "$PSScriptRoot/PostComments-Module.psm1"
              
         $newIssues = FetchAnnotatedNewIssues                
         
-        if ( ($newIssues -eq $null) -or ($newIssues.Length -eq 0))
```

Comment:
If SQ does not find any issues we may still have to resolve existing issues, so do not exit early.